### PR TITLE
make team selection scrollable

### LINKED ISSUE #1532: Teams dropdown list is not scrollable
### Current Behavior

A user with a large amount of teams or a small screen, currently can't view teams at the bottom of the list

Screenshot 2023-01-11 at 11 22 22

### Expected Behavior

Should be able to view the teams at the bottom of the list

### Steps To Reproduce

Add a large amount of teams or have a small browser window

### Environment

- FlowForge version: 1.3
- Node.js version: n/a
- npm version: n/a
- Platform/OS: macOS
- Browser: Chrome/Safari

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Description

 
i have make team drop down component scrollable 

[scrnli_17_06_2023_13-29-20.webm](https://github.com/flowforge/flowforge/assets/110285294/1a902bf5-00d3-486b-be94-62417236ab84)

## Checklist
closes #1532 

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `flowforge/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `flowforge/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: frontend/src/stylesheets/components/team-list.scss
Comment (joepavitt) [inline frontend/src/stylesheets/components/team-list.scss:1]: this is a great first step @sumitshinde-84. We try to use our existing [brand palette](https://flowforge.com/handbook/design/branding/)/variables for colours though. You can add:

```css
@import '~@flowforge/forge-ui-components/dist/scss/forge-colors.scss';
```

to the top of this `scss` file, and then pick a colour accordingly, for example, Grey 800 can be used in the `scss` file as `$ff-grey-800`

Comment (joepavitt) [reply frontend/src/stylesheets/components/team-list.scss:1]: As another comment, to practice our Iterative Improvement value, I'll definitely be okay with this once the variables are resolved. 

As a next step (for a different piece of work/PR), if you fancy the challenge, would be to _just_ have the teams scrollable, and have the "Create New Team" option fixed at the bottom of the dropdown. Feels a little strange with that as part of the scrollable list, but it's better than what we have now for when users are part of multiple teams.

**Comment by joepavitt** [comment]
@sumitshinde-84 just received a notification to re-request review with a new commit, but not seeing the new commit?

**Comment by sumitshinde-84** [comment]
> @sumitshinde-84 just received a notification to re-request review with a new commit, but not seeing the new commit?

now commit is visible ,  I'm sorry for any inconvenience caused.

**Review by joepavitt — APPROVED** [review]

**Comment by joepavitt** [comment]
All good - approved & merging - thanks @sumitshinde-84

**Comment by knolleary** [comment]
@joepavitt any reason this didn't get merged?

**Comment by joepavitt** [comment]
Good spot @knolleary - no reason

**Pull request merged**